lib/sensu-plugins-sendgrid/version.rb

```diff
@@ -1,8 +1,8 @@
 module SensuPluginsSendgrid
   module Version
-    MAJOR = 0
+    MAJOR = 1
```

Comment:
We actually do all the versioning post acceptance for several reasons:
- maintainer might disagree on how you think it should be versioned
- there is no guarantee on merge order, code review is not [FIFO](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics)) and reduces the number of times we need to tell users to rebase as a new version was released since they submitted.